docs/digger/installation/run-digger-installer.asciidoc

```diff
@@ -4,51 +4,44 @@
 
 . Clone the https://github.com/aerogear/digger-installer[Digger Installer GitHub repo]
 
-. Make a copy of the Ansible `inventory-sample` file from the repo, and update it with the information about your infrastructure.
+. Make a copy of the Ansible `inventory-sample` file from the repo, and update it with the information about your infrastructure
 +
-NOTE: For more details about the macOS Server configurations, please read the <<Setup macOS Servers>> section.
+NOTE: For a full list of variables that can be configured via the inventory file see the link:#installer-variables[Installer Variables] section.
 
+. Run the playbook using the updated `inventory-sample` and follow the instructions printed in the terminal.:
 
-. To build Android applications, you need to <<Configure the Android SDK components>>.
-//TODO: remove this step once we can configure Android SDK components via Ansible variables.
-. Run the following command with the updated `inventory-sample` and follow the instructions printed in the terminal.:
+** Command to install digger without a macOS server
++
 ----
-ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml
+ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml --skip-tags=osx-provision
 ----
 
-.The sample Ansible playbook will perform the following tasks:
---
-NOTE: All tasks are idempotent. You can run them multiple times and they will produce the same results.
---
-
-* Create a project in OpenShift
-* Install Jenkins (tag: `deploy-jenkins`)
-* Install Android SDK to a Persistent Volume (tag: `android-sdk`)
-* Configure Jenkins (tag: `configure-buildfarm`)
-* Setup macOS Servers (tag: `provision-osx`)
-
-More details about these tasks are explained in the following sections. If you are not interested in the details, skip to <<Verify Installation>>.
-
-
-=== Running Installer steps individually
-If you want to run specific tasks, enter the task tag values into the following command.
-For example, to only run the "install Jenkins" task, run:
-
-.Passing single and multiple tags to Ansible
-----
-ansible-playbook -i your-inventory-copy sample-build-playbook.yml --tags="deploy-jenkins"
-----
+** Command to install digger with a macOS server
++
 ----
-ansible-playbook -i your-inventory-copy sample-build-playbook.yml --tags="deploy-jenkins","android-sdk","configure-buildfarm"
+ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml
 ----
 
+. Carry out manual steps during the installation
++
+There are two points during the installation where there are manual steps required.
++
+.. Accepting the Android SDK Licence
+.. Copy the servers ssh key to Jenkins
```

Comment:
good to clarify which servr here maybe? 

ssh key --> public key or public ssh key